pkg/object/httppipeline/httppipeline.go

```diff
@@ -467,7 +469,10 @@ func (hp *HTTPPipeline) Handle(ctx context.HTTPContext) {
 			filterStat = lastStat
 		}()
 
-		filterIndex = hp.getNextFilterIndex(filterIndex, lastResult)
+		filterIndex, isEnd = hp.getNextFilterIndex(filterIndex, lastResult)
```

Comment:
```suggestion
		filterIndex, isEnd := hp.getNextFilterIndex(filterIndex, lastResult)
```
and remove line 446

Reply:
This symbol `:=` cannot be used here for assignment. It will create a new `filterIndex`. It will cause the outside `filterIndex` assignment to fail